### PR TITLE
delsys based on countries

### PR DESCRIPTION
**Pull request by DF5EQ**
Now you can close the former PRs and may use this one.

### CODE REVIEW

**Comment by stefantalpalaru** [comment]
Changing delsysloop_max() to always scan for DVB-T2 was done on purpose? Looks like it's used outside terrestrial broadcasts, increasing scanning time for no reason.

**Comment by DF5EQ** [comment]
The scanning of both (DVB-T and DVB-T2) was introduced in w_scan-20170107.
There the functions delsysloop_min/max were added in countries.c/.h and 
used in scan.c function initial_tune.
See commit a8a8c07eec964e06e3b649315ea191f9c8ee60c5 "merge w_scan-20170107".
So I didn't change the default behavior.
I changed the delsysloop functions to be able to set the desired looping 
(only DVB-T, only DVB-T2, both) on a per country basis instead of a 
country list basis. This gives a more fine control. I set the looping to 
only DVB-T2 for DE and AT. CO was already set in version 20170107. So if 
somebody knows the current state of the DVB systems used in different 
countries he/she is invited to add them. As default we have two loops so 
we don't miss anything.
I made the change, because it was time wasting for me to wait for the 
DVB-T loop in AT :)

Am 08.10.2018 um 19:13 schrieb Ștefan Talpalaru:

File: src/countries.c [hunk 1]
```diff
@@ -609,34 +609,28 @@ int dvbt_transmission_mode(int channel, int channellist)
...
 	}
 }
 
 /*
  * some countries don't use 2nd gen delsys yet
  */
-int delsysloop_max(int channel, int channellist)
+int delsysloop_max(int country_id)
 {
-	switch (channellist) {
```
Comment (stefantalpalaru) [inline src/countries.c:627]: Look at what gets deleted here. All these cases that were skipping the DVB-T2 scanning are now doing it by default.

**Comment by stefantalpalaru** [comment]
Merged. Thanks.

**Comment by DF5EQ** [comment]
Thanks for taking my commits.
Sorry for not answering in the last few days, but I was on a very busy 
business trip.

I think you have seen that I saved the former behavior and only changed 
the granularity for selecting the delsys loops.

Am 13.10.2018 um 00:52 schrieb Ștefan Talpalaru:

**Pull request merged**